Orbitersdk/samples/ProjectApollo/src_csm/satswitches.h

```diff
@@ -817,11 +817,17 @@ class SaturnEMSDvSetSwitch {
 	Sound &ClickSound;
 };
 
-class SaturnCabinPressureReliefLever: public ThumbwheelSwitch {
+class SaturnCabinPressureReliefLever1 : public ThumbwheelSwitch
+{
+public:
+	bool CheckMouseClickVC(int event, VECTOR3 &p);
+};
+
+class SaturnCabinPressureReliefLever2: public ThumbwheelSwitch {
```

Comment:
SaturnCabinPressureReliefLever2 can inherit from SaturnCabinPressureReliefLever1 as they use the same CheckMouseClickVC function. Also renaming this to something like GuardedPressureReliefLever would be more descriptive than Lever1 and Lever2.